.woke.yaml

```diff
@@ -1,5 +1,8 @@
 ignore_files:
   - abi-compat/*.xml
+  - doc/netplan-yaml.md
```

Comment:
This is a big hammer. But I guess there's no better way to exclude "dummy*" from verbatim code examples inside  the docs... Or maybe we should really just allow the word "dummy" globally.

This looks relevant here: https://github.com/get-woke/woke/discussions/159

I wonder if we can do something like this in our `.woke` configuration?
```
  - name: dummy
    terms:
      - dummy
    alternatives:
      - placeholder
      - sample
    severity: warning
    options:
      word_boundary_end: true
```

Well.. after some experimentation, this doesn't seem to do the trick.

Maybe instead we should change our `.woke` to use `severity: error` by default but keep it at `info` for "dummy" and use `fail-on-error: true` inside our woke GH action.

Well.. some more experimentation and this doesn't seem to work either, as `woke` doesn't differentiate between info/warning/error severity in their return value. (Really woke?!)

So in the end I think our best option is to comment out the woke rules for "dummy" (even though I rejected that idea initially). We could still keep the "wokeignore" comments around in the code, I guess. WDYT?

Reply:
I agree. If one day woke allows the use of exceptions we can put it back.